content/danielgormly/drone-plugin-kube/index.md

```diff
@@ -0,0 +1,38 @@
+---
+version: '0.1'
+date: 2019-20-05T14:54:23-07:00
+title: Kubernetes Deployment Plugin
```

Comment:
Please just name it "Kubernetes Deployment", as it's listed on the plugin index it's obviously a plugin.